eox_tagging/validators.py

```diff
@@ -314,12 +321,56 @@ def validate_equals(self, field, value):
         """
         field_value = self.instance.get_attribute(field, name=True)
 
+        if isinstance(field_value, datetime.datetime):
+            self.__compare_equal_dates(field_value, value)
+            return
+
         if not field_value:
             raise ValidationError("EOX_TAGGING | The field '{}' is required and must be equal to '{}'."
                                   .format(field, value))
         elif field_value.lower() != value.lower():
             raise ValidationError("EOX_TAGGING | The field '{}' must be equal to '{}'.".format(field, value))
 
+    def validate_between(self, field, value):
+        """
+        Function that checks that a date must be in between two dates.
+
+        Arguments:
+            - field: field to validate
+            - value: validations defined for the field
+        """
+        field_value = self.instance.get_attribute(field)
+        datetime_obj = []
+
+        for datetime_str in value:
+            try:
+                datetime_obj.append(datetime.datetime.strptime(datetime_str, DATETIME_FORMAT_VALIDATION))
+            except TypeError:
+                raise ValidationError("EOX_TAGGING | The DateTime field '{}' must follow the format '{}'."
+                                      .format(datetime_str, DATETIME_FORMAT_VALIDATION))
+
+        if field_value < datetime_obj[0] or field_value > datetime_obj[1]:
```

Comment:
In the previous commit you used the last element in this case you are using the second element, maybe I was not clear enough, but in python you can use [-1] to get the last element, https://stackoverflow.com/questions/930397/getting-the-last-element-of-a-list  if datetime_obj is ['date'], this code fails, but if you use [-1] you get the unique element